### ISSUE TITLE
Problem with group messages with emoji in title

### ISSUE THREAD

**Issue by davrsky** [shusain93/OSXMessageProxy#26]
When sending a message from AndromedaB, when I look on Messages on the mac, I see it attempted (and failed) to send the message to "aaa group chat" instead of "🙂🙂🙂 group chat"

Receiving messages still works however. 

I also see this message repeated over and over in the terminal window on the mac:
"CoreData: warning: dynamic accessors failed to find @property implementation for 'uniqueId' for entity ABCDAddressBookSource while resolving selector 'uniqueId' on class 'ABCDAddressBookSource'.  Did you remember to declare it @dynamic or @synthesized in the @implementation ?"

Not sure if related or just a harmless warning

**Comment by shusain93** [comment]
Yeah not sure how to fix this since there's no easy way to type emoji on the keyboard (which is what it's doing into the recipient field). If you copy and paste the recipient in Messages.app doesn't detect the conversation and just errors out. This, as far as I know (unless you figure out a way to type emoji on a standard keyboard in an easy, programmable way), is a limitation that can't be fixed, other than of course renaming your chat. Start a war on emoji, I guess! You of course can send emoji just fine in messages since that data is copied and pasted.

Also yeah that error is just OSX being verbose.

**Comment by davrsky** [comment]
Hmm, is it possible to programmatically select the chat from the list at the left, instead of typing the recipient name into the field?

**Comment by shusain93** [comment]
Not as far as I know. You can technically use accessibility and find the nearest match by scanning through the whole list, scrolling, and then trying again. You used to be able to but apple removed a ton of the scripting powers because people were using it for spam.
